### PR TITLE
added view history call

### PR DESCRIPTION
**Pull request by jenjenp**

### CODE REVIEW

File: services.yaml [hunk 1]
```diff
@@ -2268,6 +2268,32 @@ paths:
           $ref: "#/responses/UnauthorizedError"
         404:
           $ref: "#/responses/NotFoundError"
+  /services/{env_id}/views/{view_id}/history:
+    get:
+      tags:
+        - Views
+      summary: Fetch View History
+      description: Fetch View History
```
Comment (chook) [inline services.yaml]: more description

File: services.yaml [hunk 2]
```diff
@@ -3387,6 +3413,35 @@ definitions:
         items:
           type: string
           example: MyApp
+  ViewHistory:
+    type: object
+    properties:
+      created_on:
+        type: string
+        description: View creation time. Date in ISO8601 format (yyyy-MM-ddTHH:mm:ss.SSSZZ)
+        example: 2018-02-28T22:47:09.038+02:00
```
Comment (chook) [inline services.yaml:3422]: didn't we move to UTC only dates with Z at the end?

File: services.yaml [hunk 3]
```diff
@@ -3387,6 +3413,35 @@ definitions:
...
+        example: 2018-05-01T10:00:00.020+03:00
+      update_method:
+        type: string
+        enum: [manual,api]
+        description: View update method
+        example: manual
+      updated_by:
+        type: string
+        description: updated by
+        example: John Doe #do we want to see the udf name?
```
Comment (chook) [inline services.yaml]: there's a comment here. it's fine to keep it as "John Doe" for now and worry about the UDF later if at all

**Review by chook — CHANGES_REQUESTED** [review]

File: services.yaml (hunk 1 above)
Comment (jenjenp) [reply services.yaml]: added

File: services.yaml (hunk 3 above)
Comment (jenjenp) [reply services.yaml]: comment removed

File: services.yaml (hunk 2 above)
Comment (jenjenp) [reply services.yaml:3422]: fixed

**Review by chook — APPROVED** [review]

**Pull request merged**